File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.cc
#include <iostream>
using namespace std;

int main(void)
{
	int student[30] = { 0 };
	for (int i = 0; i < 28; i++) {
		int n;
		cin >> n;
		student[n - 1] = 1;
	}

	for (int i = 0; i < 30; i++) {
		if (student[i] == 0) cout << i + 1 << endl;
	}
}

File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);

        boolean student[] = new boolean[30];
        for(int i=0; i<28; i++) {
            int num = s.nextInt();
            student[num-1] = true;
        }
        for (int i=0; i<30; i++) {
            if(!student[i]) {
                System.out.println(i+1);
            }
        }

        s.close();
    }
}